### PR TITLE
sc_vfs: improve read command output

### PR DESCRIPTION
**Pull request by vincent-d**
This improve the output of the read command at the end of the read.

Before:

```
2016-09-08 13:10:44,007 - INFO # > vfs r /test.txt 30
2016-09-08 13:10:44,015 - INFO # 00000000: 3132 3334 3536 3738 3938 3735 3436 3534  1234567898754654
2016-09-08 13:10:44,018 - INFO # 00000010: 3531 3332  5132
2016-09-08 13:10:44,019 - INFO # -- EOF --
```

After:

```
2016-09-08 13:16:36,392 - INFO # > vfs r /test.txt 30
2016-09-08 13:16:36,401 - INFO # 00000000: 3132 3334 3536 3738 3938 3735 3436 3534  1234567898754654
2016-09-08 13:16:36,411 - INFO # 00000010: 3531 3332                                5132............
2016-09-08 13:16:36,413 - INFO # -- EOF --
```

### CODE REVIEW

File: sys/shell/commands/sc_vfs.c [hunk 1]
```diff
@@ -200,9 +201,16 @@ static int _read_handler(int argc, char **argv)
...
+            if ((k % 2) == 0) {
+                putchar(' ');
+            }
+            putchar(' ');
+            putchar(' ');
+        }
         putchar(' ');
         putchar(' ');
-        for (int k = 0; k < res; ++k) {
+        for (int k = 0; k < sizeof(buf); ++k) {
```
Comment (jnohlgard) [inline sys/shell/commands/sc_vfs.c:213]: are the trailing dots really desired?
In my opinion the corrected padding was all that is needed. `xxd` on Linux for example does not add any trailing dots.

**Comment by jnohlgard** [comment]
Thanks for the fix!

File: sys/shell/commands/sc_vfs.c (hunk 1 above)
Comment (vincent-d) [reply sys/shell/commands/sc_vfs.c:213]: Indeed, that's not needed,  `k < res` is better then.

**Comment by jnohlgard** [comment]
I have incorporated the changes in the latest revision of RIOT-OS/RIOT#4616, squashed into the other VFS commit

**Pull request closed without merging**